.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Feature request
+about: Suggest an idea for the qonnx-frontend
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+## Prerequisites
+Please talk to either us before creating a new feature request. So that you can check that the idea is not already in active development.
```

Comment:
```suggestion
Please talk to us before creating a new feature request. So that you can check that the idea is not already in active development.
```